### DISCUSSION TITLE
class support ?

### DISCUSSION THREAD

**Discussion by erikpa1** [Aleph-Alpha/ts-rs#149] [General]
Hi, is it possible, to add support for classes  (also with default  attributes value )?  not only interfaces ?

**Comment by vknowles-rv** [comment]
A class is a piece of code that can be instantiated. Instantiation means executable code. That seems beyond the scope of this package since its intention is type checking and type safety.

**Reply by erikpa1** [reply]
ok, I will do it what JavaScript people does, create my own package. Have a nice day :D
